packages/solid/src/use-chat.ts

```diff
@@ -54,15 +62,111 @@ export type UseChatHelpers = {
   input: Accessor<string>;
   /** Signal setter to update the input value */
   setInput: Setter<string>;
+  /** An input/textarea-ready onChange handler to control the value of the input */
+  handleInputChange: (
+    e:
+      | (Event & { currentTarget: HTMLInputElement })
+      | (Event & { currentTarget: HTMLTextAreaElement }),
```

Comment:
is this the same as for react? it's important for me to keep things aligned (even if there could be more functionality by expanding), and then expand the functionality for all providers.

Reply:
I'm probably not understanding the question, but this is the Solid-equivalent of the type in React afaik. Solid doesn't provide specific Event types, you just add `currentTarget` and `target` to the generic `Event` type

Reply:
https://www.solidjs.com/guides/typescript#event-handlers

Never mind, I could use these types above and will in the next commit